pallets/automation-time/src/lib.rs

```diff
@@ -1447,4 +1447,16 @@ pub mod pallet {
 			T::ExecutionWeightFee::get().saturating_mul(weight_as_balance)
 		}
 	}
+
+	impl<T: Config> pallet_valve::Shutdown for Pallet<T> {
+		fn is_shutdown() -> bool {
+			Pallet::<T>::is_shutdown()
```

Comment:
is it Pallet::<T>::is_shutdown() or is it Self::is_shutdown? most of our other functions to get DB storage is querying the function off of `Self::` rather than `Pallet::<T>::`. although could just be the same syntax, prob good to keep consistency